WordPress/gutenberg
# Simplify and document running `phpcs` locally

This PR makes it easier to run `phpcs` locally.

### To test

- Make sure the Travis build, especially the `phpcs` step, still passes.
- Install `phpcs` following the instructions in the comments of `bin/run-phpcs.sh`.  Run the script and verify that it works (no output, exit code `0`).
